### ISSUE TITLE
Calendar KeyBoard Accessibility Issue When Showed Two Months And Pressed the ArrowRight

### ISSUE THREAD

**Issue by janey-ciandt** [primefaces/primereact#7604]
### Describe the bug

sandbox link：https://stackblitz.com/edit/re2gxksh?file=src%2FApp.jsx

when we keep pressing the ArrowRight,the console will show an error like that: 
Uncaught TypeError: Cannot set properties of null (setting 'tabIndex')
    at navigateToMonth2 (primereact_calendar.js?v=8ce8adc0:6442:28)
    at onDateCellKeydown2 (primereact_calendar.js?v=8ce8adc0:6343:11)
    at onKeyDown (primereact_calendar.js?v=8ce8adc0:8191:16)
    at HTMLUnknownElement.callCallback2 (chunk-QXLG2TGQ.js?v=95b925d2:3678:22)
    at Object.invokeGuardedCallbackDev (chunk-QXLG2TGQ.js?v=95b925d2:3703:24)
    at invokeGuardedCallback (chunk-QXLG2TGQ.js?v=95b925d2:3737:39)
    at invokeGuardedCallbackAndCatchFirstError (chunk-QXLG2TGQ.js?v=95b925d2:3740:33)
    at executeDispatch (chunk-QXLG2TGQ.js?v=95b925d2:7044:11)
    at processDispatchQueueItemsInOrder (chunk-QXLG2TGQ.js?v=95b925d2:7064:15)
    at processDispatchQueue (chunk-QXLG2TGQ.js?v=95b925d2:7073:13)

### System Information

```Shell
Ubuntu 21.0
Google Chrome
```
